permissions/jolla-messages.profile

```diff
@@ -16,5 +16,6 @@
 private-bin jolla-messages
 # FIXME: jolla-messages has odd app service name?
```

Comment:
You could also comment here that this is legacy that will be removed at some point. Not necessary but I think it would be more useful than the current FIXME comment